babel.config.js

```diff
@@ -35,12 +35,15 @@ module.exports = api => {
           // Convert ES import/export syntax during tests, since Jest expects
           // CommonJS, not ES import/export syntax
           modules: api.env("test") ? "cjs" : false,
+          targets: {
```

Comment:
this is the only non-trailing comma change in this file

Reply:
Nit: Maybe add a comment here explaining that this is to support IE 11.

Reply:
Good point will do in the next pr!